react/features/video-quality/components/VideoQualitySlider.web.js

```diff
@@ -318,25 +312,31 @@ class VideoQualitySlider extends Component {
      * component's slider options.
      *
      * @private
-     * @returns {void}
+     * @returns {number}
      */
     _mapCurrentQualityToSliderValue() {
         const { _audioOnly, _receiveVideoQuality } = this.props;
         const { _sliderOptions } = this;
 
         if (_audioOnly) {
-            const audioOnlyOption = _sliderOptions.find(
-                ({ audioOnly }) => audioOnly);
+            const audioOnlyOption: ?SliderOption = _sliderOptions.find(
+                (option: SliderOption) => option.audioOnly);
 
-            return _sliderOptions.indexOf(audioOnlyOption);
+            return audioOnlyOption
```

Comment:
can't this be written as concise as before?

Reply:
Looks like they can be written like they were before. I couldn't make heads or tails of some flow errors and thought this solved one, but looks like the issue was actually solved by the other flow-related changes in the file.

Reply:
So I think what's happening is in the above flow thinks the .find can return null. And then indexOf shouldn't be called with potentially null and flow doesn't like that. I don't know the flow syntax to get around that.